test/controllers/microposts_controller_test.rb

```diff
@@ -47,14 +47,15 @@ class MicropostsControllerTest < ActionDispatch::IntegrationTest
   # Test update.
 
   test 'should succeed patch update micropost' do
-    patch micropost_path(@micropost), params: 
-    { 
-      micropost: 
-      { 
-        content: @micropost.content, 
-        user_id: @micropost.user_id
-      }, format: :html
-    }
+    patch micropost_path(@micropost),
+          params:
+          {
+            micropost:
+            {
+              content: @micropost.content,
+              user_id: @micropost.user_id
+            }, format: :html
```

Comment:
move `format: :html` to the same indent level as `Micropost:`. On second thought, you can probably just delete it. Remember, we figured out it was a different bug.